media-sound/kid3/patches/kid3-3.9.5.patchset

```diff
@@ -0,0 +1,51 @@
+From 18de459335880bcf3cd5d4c212c94dcba7d04ff4 Mon Sep 17 00:00:00 2001
+From: Schrijvers Luc <begasus@gmail.com>
+Date: Wed, 22 May 2024 09:12:16 +0200
+Subject: Add Haiku specific paths
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ddb7afc..c29405f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -140,6 +140,14 @@ elseif(ANDROID)
+   set(WITH_PLUGINSDIR_DEFAULT ".")
+   set(QT_ANDROID_APP_VERSION ${KID3_VERSION})
+   set(QT_ANDROID_APP_VERSION_CODE 29)
++elseif(HAIKU)
++  set(WITH_DATAROOTDIR_DEFAULT "data")
++  set(WITH_DOCDIR_DEFAULT "data/doc/kid3-qt")
```

Comment:
If you are setting it manually anyway, you could set it correctly, i.e. "documentation/kid3-qt" (or "documentation/packages/kid3-qt" if you want to mimic $docDir).

Reply:
I could change it here, patch already merged upstream for this. :)

Reply:
Updated, seems the documentation isn't installed either way. :)